modules/dbnd/src/dbnd/_core/task_run/task_run_sync_local.py

```diff
@@ -64,7 +64,6 @@ def _sync_input_or_output(self, param_definition, old_target, new_target):
 
         else:
             self.inputs_to_sync.append((param_definition, old_target, new_target))
-            setattr(self.task, param_definition.name, new_target)
```

Comment:
Why remove it? Looks like a bug to me

Reply:
Its a duplicated `setattr`, there's another one that does the same thing in line 99